docs/contributing/01-start-here/05-development.md

```diff
@@ -35,6 +46,8 @@ cd wing
 pnpm install
 ```
 
+![Bird High Five](https://i.giphy.com/media/TgrXouvj1MwKoREXap/200w.gif)
```

Comment:
I really love the gif, though reviewing the page it kinda looks strange by itself
```suggestion

```

Reply:
yeah makes sense. saw the gif on the main page and wanted to add more. but i think it doesnt fit the tone of the rest of the content.